cpp/src/Ice/WSEndpoint.h

```diff
@@ -39,7 +39,10 @@ class WSEndpoint : public EndpointI, public std::enable_shared_from_this<WSEndpo
     virtual bool secure() const;
 
     virtual TransceiverPtr transceiver() const;
-    virtual void connectors_async(Ice::EndpointSelectionType, const EndpointI_connectorsPtr&) const;
+    virtual void connectorsAsync(
+        Ice::EndpointSelectionType,
+        std::function<void(const std::vector<IceInternal::ConnectorPtr>&)>,
```

Comment:
I would remove the IceInternal qualification - everything is in IceInternal here.